src/config/config.cc

```diff
@@ -575,6 +577,19 @@ void Config::initFieldCallback() {
              if (!srv) return Status::OK();
              return srv->GetNamespace()->LoadAndRewrite();
            }},
+          {"minor-columns-write-buffer-size",
+           [this](Server *srv, const std::string &k, const std::string &v) -> Status {
+             if (!srv) return Status::OK();
+             const std::vector<ColumnFamilyID> column_families = {kColumnFamilyIDZSetScore, kColumnFamilyIDPubSub,
+                                                                  kColumnFamilyIDPropagate, kColumnFamilyIDStream,
+                                                                  kColumnFamilyIDSearch};
+             for (const auto &cf : column_families) {
+               auto s = srv->storage->SetOptionForColumnFamily(cf, "write_buffer_size",
+                                                               std::to_string(minor_columns_write_buffer_size * KiB));
```

Comment:
Good suggestion.

Reply:
Do you mean prefix this option with rocksdb，like `rocksdb.minor-columns-write-buffer-size` ?

I'm struggling with this. Because all options prefix with `rocksdb` correspond to the options of rocksdb, but this is a behavior of kvrocks.

Reply:
> Because all options prefix with rocksdb correspond to the options of rocksdb, but this is a behavior of kvrocks.

`rocksdb.write_buffer_size` previously is also a part of rocksdb and written inside `config.rocks_db`, maybe we should regard them as same thing?